centreon-archive/centreon-archived
# enh(UI): PENDO - Add a unique identifier for css objects

## Description

Add a unique identifier for css objects in RS


**Fixes** # (issue)


RS - LISTING

- ACK / DT / Check buttons

- ACK / DT / Check icons on each resources line

- Url / Action note icons on each resources line

- Clear filter icon

- Manage filters icon

RS - DETAIL PANEL

- When user click on “Services” tab  

- When user click on “Details” tab  

- When user click on “Graph” tab 

- When user click on “Timeline” tab 

TAB GRAPH OR HEADER PANEL

- When user click on “Export” button

- When user click on “As Displayed” / “Medium Size” / “Small Size” buttons

- When user click on “Go to performance page”

- When user click on “Display Events” // ASAP

- Selection of limit of line per page // ASAP

- When user click on “1 Day” / “7 Days” / “31 Days” buttons  

- When user click on “From / To” selector  

- When user click on “Graph option” icon 

- When user click on “Copy” command button  

- When user click on “Edit service” icon or “Edit host” icon (same id whatever the kind of resource)  

- When user click on “Copy the link”  

- When user click on “shortcut”   

- When user click on “View logs” button  

- When user click on “View report” button  

Poller Menu

- When user click to extend / reduce “Pollers” status :check_mark: 

- When user click on “Configure Pollers” button :check_mark: 

- When user click on “Export Configuration button” :check_mark: 

Top Counter HOST/SERVICE

- When user click to extend / reduce Resource (Host and Services) status 

- When user clicks on status “pastille” (Warning, Critical, OK, …) 

- When user clicks on labels in extend menu (Warning, critical, …) 

## Type of change


## How to test

On Element click right on select "inspect" , you must see on console , for each HTML element,CSS object an attribute name like data-testid who return a value.



## Target serie

- [x] 20.10.x
- [x] 22.04.x (master)
